### PR TITLE
Add test for group join request with Unicode reason

### PR DESCRIPTION
**Pull request by rra**
This previously failed when generating the email message with an
encoding error.  It now works, probably because Grouper is pure
Python 3.

### CODE REVIEW

File: itests/fe/group_join_test.py [hunk 1]
```diff
@@ -145,3 +147,17 @@ def test_group_join_group_as_owner(tmpdir: LocalPath, setup: SetupTest, browser:
...
+
+def test_request_join_unicode(tmpdir: LocalPath, setup: SetupTest, browser: Chrome) -> None:
+    with setup.transaction():
+        setup.add_user_to_group("gary@a.co", "some-group", "owner")
+
+    with frontend_server(tmpdir, "cbguder@a.co") as frontend_url:
+        browser.get(url(frontend_url, "/groups/some-group/join"))
+        page = GroupJoinPage(browser)
+
+        page.set_reason("защото причини")
```
Comment (wrodriguez-dbx) [inline itests/fe/group_join_test.py]: изглежда законно
